doc/subsystems/logging/logger.rst

```diff
@@ -347,6 +354,29 @@ the message), and severity level. Once all backends are iterated, the message
 is considered processed by the logger, but the message may still be in use by a
 backend.
 
+.. _logger_strings:
+Logging strings
```

Comment:
add a blank line after the label directive:

    .. _logger_strings:

    Logging strings
    ===============

This change will fix the other error noted in the CI doc build test.

Reply:
done